docker_images/paddlenlp/requirements.txt

```diff
@@ -0,0 +1,6 @@
+starlette==0.14.2
+api-inference-community==0.0.23
+huggingface_hub==0.5.1
+paddlepaddle==2.3.2
+## TODO: switch to a release when it's out
+git+https://github.com/PaddlePaddle/PaddleNLP
```

Comment:
Could we pin to a specific commit? That way we'll have certainty of what's released

Reply:
this is temporary, just to get the unit test passed.
We'll do a release of PaddleNLP before we finalized this PR and lock this to a version